src/hooks/pagination/pagination.tsx

```diff
@@ -1,12 +1,12 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, useCallback } from "react";
 import { useLazyQuery, useApolloClient } from "@apollo/client";
 
 function getStartValue(startingValue: number, limit: number, prev: boolean): number {
 	if (!prev) {
 		return startingValue;
 	}
 
-	if (startingValue % limit == 0) {
+	if (startingValue % limit === 0) {
```

Comment:
what is `0`?

Reply:
let us consider we have limit of 10 and we are at the end and we have 7 items, them the limit becones 7. That is why i am checking whether startingValue is divisible by limit or not

Reply:
> let us consider we have limit of 10 and we are at the end and we have 7 items, them the limit becones 7. That is why i am checking whether startingValue is divisible by limit or not

for this can't we simply check like this 
` (startingValue< limit) {
          limit= startingValue
}`

Reply:
I have added comments